.pre-commit-config.yaml

```diff
@@ -83,17 +83,20 @@ repos:
           - stylelint-order@4.1.0
         args: ["--fix"]
 
-  - repo: https://github.com/errata-ai/vale
-    rev: v3.8.0
+  - repo: https://github.com/executablebooks/mdformat
+    rev: 0.7.18
     hooks:
-      - id: vale
-        args: [--config=.vale/.vale.ini]
-        # There's no way to automatically convert vale suggestions/warnings to errors, and so they won't appear at all unless
-        # there's an error raised.
-        # pre-commit's verbose mode means that suggestions and warnings are always shown even if there's no error raised.
-        # See https://github.com/errata-ai/vale/issues/575.
-        verbose: true
-
+      - id: mdformat
+        args:
+          [
+            --ignore-missing-references,
+            --wrap=no,
```

Comment:
```suggestion
          [
            --help,
            --ignore-missing-references,
            --wrap=no,
```

Reply:
Hi @huong-li-nguyen I didn't include it because I didn't understand how `--help` would work, doesn't it just print help text for us? And I'm not seeing a pre-commit fail now, all is passing, AFAICT? Are you seeing something different?

Reply:
Ah yes, now everything is passing!

So when I previously checked it the pre-commit step for the mdformat just failed but it didn't fix the files automatically. When I added the `--help` tag it automatically fixed the files, but now I am confused 😄 Might be good to introduce an error into one of the md files and see if it fixes with or without the `--help` tag

Reply:
The `--help` shouldn't make a difference here. I think what you are seeing here is that for reasons I don't understand mdformat seems slightly unpredictable about when it decides to fix things. But it seems to be working well enough without the `--help`.